### DISCUSSION TITLE
What's the lowest android version this can support technically?

### DISCUSSION THREAD

**Discussion by IamSanjid** [fzyzcjy/flutter_rust_bridge#1770] [Q&A] [answered]
It probably depends on what type of dependencies I use in the rust part right? But if you had to do some educational guess what would it be? Any suggestion regarding this case? Flutter supports as old as 4.4 android oh boi that's so old I remeber using them back in 2013..

**Comment by fzyzcjy** [comment, marked as answer]
flutter_rust_bridge does not require any specific android version IMHO. So I agree with your guess - just look at flutter + rust crate's requirements. But I did not do experiments to really check android 4.4, and you can also try it and see what happens!
